scripts/quickstart.sh

```diff
@@ -168,7 +175,7 @@ sleep 0.5
 # Start to query nodes.
 for i in `seq 1 2`
 do
-  ./thanos query \
+  $THANOS_EXECUTABLE query \
```

Comment:
One more thing: Technically `${...}` is the pattern for variables. Can we be consistent here and on other variables you define here?